### PR TITLE
build(deps) npm update + shadow-bump of commitizen

### PR DESCRIPTION
**Pull request by louis-bompart**
Issue: https://github.com/coveo/cli/security/dependabot/package-lock.json/merge/open

Diagnostic:
```
> npm ls merge
`-- cz-conventional-changelog@3.3.0
  `-- commitizen@4.2.3
    `-- find-node-modules@2.0.0
      `-- merge@1.2.1
```

Resolution: https://github.com/commitizen/cz-cli/pull/824

On our end, we just 'update' the sub dep (it's janky: install the dep version you want, then uninstall it. The pkg-lock tho will keep the version you installed).

This makes so we have our dependency tree updated and don't have to wait for `cz-conventional-changelog` to be bumped.

Also, bonus round: I ran an `npm update` to dust off our deps.

Edit: Bonus round 2. Updated to prettier 2.3.0
https://prettier.io/blog/2021/05/09/2.3.0.html

Some minor changes, reapplied prettier to the project to keep it tidy

-----
CDX-227

### CODE REVIEW

File: packages/cli/src/commands/org/list.spec.ts [hunk 1]
```diff
@@ -24,9 +24,9 @@ describe('org:list', () => {
 
   mockedClient.mockImplementation(
     () =>
-      (({
+      ({
         getAllOrgsUserHasAccessTo: mockGetOrgs,
-      } as unknown) as AuthenticatedClient)
+      } as unknown as AuthenticatedClient)
```
Comment (louis-bompart) [inline packages/cli/src/commands/org/list.spec.ts:29]: prettier has spoken.

File: packages/cli/src/hooks/prerun/prerun.spec.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('hooks:prerun', () => {
   const mockSet = jest.fn();
 
   mockConfig.mockImplementation(
-    () => (({get: mockGet, set: mockSet} as unknown) as Config)
+    () => ({get: mockGet, set: mockSet} as unknown as Config)
```
Comment (louis-bompart) [inline packages/cli/src/hooks/prerun/prerun.spec.ts:14]: prettier has spoken.

File: packages/cli/src/commands/config/set.spec.ts [hunk 3]
```diff
@@ -16,16 +16,16 @@ describe('config:set', () => {
...
+      } as unknown as Config)
   );
 
   mockedClient.mockImplementation(
     () =>
-      (({
+      ({
         getUserHasAccessToOrg: mockGetHasAccessToOrg,
-      } as unknown) as AuthenticatedClient)
+      } as unknown as AuthenticatedClient)
```
Comment (louis-bompart) [inline packages/cli/src/commands/config/set.spec.ts:28]: prettier has spoken.

File: packages/cli/src/commands/config/get.spec.ts [hunk 4]
```diff
@@ -13,10 +13,10 @@ describe('config:get', () => {
   beforeEach(() => {
     mockedConfig.mockImplementation(
       () =>
-        (({
+        ({
           get: mockGet,
           set: mockSet,
-        } as unknown) as Config)
+        } as unknown as Config)
```
Comment (louis-bompart) [inline packages/cli/src/commands/config/get.spec.ts:19]: prettier has spoken.

File: packages/cli/src/commands/auth/login.spec.ts [hunk 5]
```diff
@@ -52,10 +52,10 @@ describe('auth:login', () => {
 
     mockedConfig.mockImplementation(
       () =>
-        (({
+        ({
           get: mockConfigGet,
           set: mockConfigSet,
-        } as unknown) as Config)
+        } as unknown as Config)
```
Comment (louis-bompart) [inline packages/cli/src/commands/auth/login.spec.ts:58]: prettier has spoken.

File: packages/cli/src/commands/auth/login.spec.ts [hunk 6]
```diff
@@ -35,10 +35,10 @@ describe('auth:login', () => {
   beforeEach(() => {
     mockedAuthenticatedClient.mockImplementation(
       () =>
-        (({
+        ({
           getAllOrgsUserHasAccessTo: mockListOrgs,
           getUserHasAccessToOrg: mockGetHasAccessToOrg,
-        } as unknown) as AuthenticatedClient)
+        } as unknown as AuthenticatedClient)
```
Comment (louis-bompart) [inline packages/cli/src/commands/auth/login.spec.ts:41]: prettier has spoken.

**Review by olamothe — APPROVED** [review]

**Review by y-lakhdar — APPROVED** [review]

**Pull request merged**